aspnetcore/signalr/diagnostics.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Diagnostics
+author: anurse
+description: Learn how to gather diagnostics from your SignalR application.
+monikerRange: '>= aspnetcore-2.1'
+ms.author: anurse
+ms.custom: signalr
+ms.date: 02/27/2019
+uid: signalr/diagnostics
+---
+# Diagnostics
+
+By [Andrew Stanton-Nurse](https://twitter.com/anurse)
+
+This article provides guidance for gather diagnostics from your SignalR application to help troubleshoot issues.
+
+## Server-Side Logging
+
+> [!NOTE]
+> Server-side logs may contain sensitive information from your application. **Never** post raw logs from production applications to public forums like GitHub.
+
+Since SignalR is part of ASP.NET Core, it uses the ASP.NET Core Logging system. In the default configuration, SignalR logs very little information, but this can configured. See the documentation on [ASP.NET Core Logging](xref:fundamentals/logging/index#configuration) for details on configuring ASP.NET Core logging.
```

Comment:
```suggestion
Since SignalR is part of ASP.NET Core, it uses the ASP.NET Core logging system. In the default configuration, SignalR logs very little information, but this can configured. For details on configuring ASP.NET Core logging, see the documentation on [ASP.NET Core Logging](xref:fundamentals/logging/index#configuration).
```